### PR TITLE
Adding docs for calico MTU

### PR DESCRIPTION
**Pull request by rifelpet**
This was cherry-picked from https://github.com/kubernetes/kops/pull/7309 The code changes were already implemented in other PRs but the docs addition is still useful so I'm splitting it out here.

### CODE REVIEW

**Comment by hakman** [comment]
Looks good

File: docs/networking.md [hunk 1]
```diff
@@ -218,6 +218,18 @@ When you enable cross-subnet mode in kops, an addon controller ([k8s-ec2-srcdst]
 will be deployed as a Pod (which will be scheduled on one of the masters) to facilitate the disabling of said source/destination address checks.
 Only the masters have the IAM policy (`ec2:*`) to allow k8s-ec2-srcdst to execute `ec2:ModifyInstanceAttribute`.
 
+### Configuring Calico MTU
+
+The Calico MTU is configurable by editing the cluster and setting `mtu` option in the calico configuration.
+AWS VPCs support jumbo frames, so on cluster creation kops sets the calico MTU to 8912 bytes (9001 minus overhead).
```
Comment (hakman) [inline docs/networking.md]: Maybe add a link to the docs:
https://docs.projectcalico.org/networking/mtu#determine-mtu-size

Comment (rifelpet) [reply docs/networking.md]: done

**Pull request merged**